client/me/privacy/dpa.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Card } from '@automattic/components';
+import SectionHeader from 'calypso/components/section-header';
+import { useDispatch } from 'react-redux';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import wp from 'calypso/lib/wp';
+
+const DPA = () => {
+	const [ isLoading, setLoading ] = React.useState( false );
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const requestDpa = async () => {
+		try {
+			setLoading( true );
+			await wp.req.post( '/me/request-dpa', { apiNamespace: 'wpcom/v2' } );
+			dispatch(
+				successNotice(
+					translate( 'Request successful! We are sending you our DPA via email', {
+						comment:
+							'A Data Processing Addendum (DPA) is a document to assure customers, vendors, and partners that their data handling complies with the law.',
+					} ),
+					{ id: 'request-dpa-notice' }
```

Comment:
Nit: Should we declare that above so it's reused for both notices?

Reply:
Does the `id` need to be there at all? All it does is that when I press the button and make the request repeatedly, the old notice will be replaced instead of multiple ones being shown. A very minor difference.

Reply:
I appreciate the practice of intentionally not duplicating notices, so I'd like to keep the `id` if possible.

Reply:
Looping back to the original comment, here's the fix: https://github.com/Automattic/wp-calypso/pull/53452/commits/9ba1290074e21b26565e0ccaf2bcd698822dce47

Reply:
ok ok, in that case I wish notices could do that automatically somehow, with auto-assigned IDs. The use case is quite frequent, I would even say it's the desired default. I don't know if it can be implemented at all though.